README.md

```diff
@@ -11,18 +11,19 @@ Goals
   3. Minimal changes
   4. Avoiding _scope creep_ which causes unnecessary changes to accumulate
 
-Constaints
+Constraints
 --------------------------------------------------------------------------------
 
 For now:
 
-  1. Damage changes only, no messing with other variables like falloffs or headshot multipliers
-  2. SMG, R201/101, Flatline, EVA-8 and Spitfire changes only
-  3. No tactical and ordnance changes (yet)
```

Comment:
Ordnance change already made for testing

Reply:
Removed line